altair/vegalite/v5/theme.py

```diff
@@ -28,11 +20,47 @@
     else:
         from typing_extensions import TypeAlias
 
-P = ParamSpec("P")
+    from altair.utils.plugin_registry import PluginEnabler
+
+
 AltairThemes: TypeAlias = Literal["default", "opaque"]
 VEGA_THEMES: list[LiteralString] = list(get_args(VegaThemes))
 
 
+# HACK: See for `LiteralString` requirement in `name`
+# https://github.com/vega/altair/pull/3526#discussion_r1743350127
+class ThemeRegistry(PluginRegistry[Plugin[ThemeConfig], ThemeConfig]):
+    def enable(
+        self,
+        name: LiteralString | AltairThemes | VegaThemes | None = None,
+        **options: Any,
+    ) -> PluginEnabler:
+        """
```

Comment:
# NOTE
Previously the only thing in https://github.com/vega/altair/pull/3618/files#diff-43d26f1359b1c241f9f632a56cbc8cef42f2fee40941e284620ebefc3de0b25e